openedx/features/classrooms/tests.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
Let's move this file in a separate `tests` directory, as is followed in other features.

Reply:
Done.